appveyor.yml

```diff
@@ -67,3 +67,5 @@ test_script:
  - '%DC% --version'
  - make -f win32.mak DFLAGS=-g rdmd dustmite ddemangle changed
  - make -f win32.mak DFLAGS=-g test
+ - make -f win64.mak DFLAGS=-g rdmd dustmite ddemangle changed
```

Comment:
@CyberShadow Is this the way to test the 64-bit version?

Reply:
It is a way, I think that should work.

Reply:
Hmm, actually AppVeyor doesn't seem to be enabled for this repository?

@wilzbach Do you know?

Reply:
We could use Github actions too, that might be easier to setup than AppVeyor nowadays.

Reply:
Theoretically AppVeyor would just be flipping a switch.

Reply:
Seems to work! https://ci.appveyor.com/project/CyberShadow/tools

For some reason it put my GitHub username in the URL, even though in the settings I can see that it is linked to dlang/tools. It created a commit status successfully too.

Please rebase to trigger it for this PR?